drivers/kscan/Kconfig

```diff
@@ -18,10 +18,4 @@ module = KSCAN
 module-str = kscan
 source "subsys/logging/Kconfig.template.log_config"
 
-config KSCAN_INIT_PRIORITY
-	int "Keyboard scan driver init priority"
-	default 40
-	help
-	  Keyboard scan device driver initialization priority.
-
```

Comment:
Why removing completely KSCAN_INIT_PRIORITY and not, for example, change its default to APPLICATION_INIT_PRIORITY? In this way it is possible to configure it if required by some application

Reply:
changed value and removed some custom defaults that are no longer needed.